src/coordinator/services/m3coordinator/server/server.go

```diff
@@ -215,7 +219,53 @@ func Run(runOpts RunOptions) {
 		}, nil)
 	}
 
-	handler, err := httpd.NewHandler(fanoutStorage, executor.NewEngine(fanoutStorage),
+	var (
+		namespaces  = clusters.ClusterNamespaces()
+		downsampler downsample.Downsampler
+	)
+	if n := namespaces.NumAggregatedClusterNamespaces(); n > 0 {
```

Comment:
nit: Pull this out into a function?

Reply:
Hm, this is a function `NumAggregatedClusterNamespaces()`?

Reply:
Sorry, meant the whole block that builds the downsampler not specifically this line haha

Reply:
Sure thing, done.